js/huobi.js

```diff
@@ -3018,4 +3019,64 @@ module.exports = class huobi extends Exchange {
             }
         }
     }
+
+    async fetchFundingRateHistory (symbol = undefined, since = undefined, limit = undefined, params = {}) {
+        //
+        // Gets a history of funding rates with their timestamps
+        //  (param) symbol: Future currency pair
+        //  (param) limit: not used by huobi
+        //  (param) since: not used by huobi
+        //  (param) params: Object containing more params for the request
+        //  return: [{symbol, fundingRate, timestamp, dateTime}]
+        //
+        await this.loadMarkets ();
+        const request = {};
+        if (symbol === undefined) {
+            throw new ArgumentsRequired (this.id + ' fetchFundingRateHistory() requires a symbol argument');
+        } else {
+            const market = this.market (symbol);
+            request['contract_code'] = market['id'];
```

Comment:
@kroitor We were having some trouble finding a symbol that worked for this method, we had to use a huobi symbol (e.g.  'BTC-USDT'), comment out the 2 lines above and replace them with

```
// const market = this.market (symbol);
// request['contract_code'] = market['id'];
request['contract_ code'] = symbol;
```

to get this to work

If we tried any of the symbols from `huobi symbols`, we ended up getting a contract doesn't exist error

```
% huobi fetchFundingRateHistory ZRX/USD:ZRX
2021-12-08T06:12:40.265Z
Node.js: v14.17.0
CCXT v1.63.19
huobi.fetchFundingRateHistory (ZRX/USD:ZRX)
ExchangeError huobi {"status":"error","err_code":1014,"err_msg":"This contract doesnt exist.","ts":1638943962611}
---------------------------------------------------
[ExchangeError] huobi {"status":"error","err_code":1014,"err_msg":"This contract doesnt exist.","ts":1638943962611}

    at handleErrors               ../../ccxt/ccxt/js/huobi.js:3017               throw new ExchangeError (feedback);                                             
    at                            ../../ccxt/ccxt/js/base/Exchange.js:670        this.handleErrors (response.status, response.statusText, url, method, responseH…
    at processTicksAndRejections  internal/process/task_queues.js:95                                                                                             
    at async timeout              ../../ccxt/ccxt/js/base/functions/time.js:203  return await Promise.race ([promise, expires.then (() => { throw new TimedOut (…
    at fetchFundingRateHistory    ../../ccxt/ccxt/js/huobi.js:3039               const response = await this.contractPublicGetLinearSwapApiV1SwapHistoricalFundi…
    at async main                 ../../ccxt/ccxt/examples/js/cli.js:246         const result = await exchange[methodName] (... args)                            

huobi {"status":"error","err_code":1014,"err_msg":"This contract doesnt exist.","ts":1638943962611}
```

this is with a defaultType of `swap`